hack/quick-install.sh

```diff
@@ -32,19 +32,18 @@ EOF
 bash cmd/operator/generate.sh
 helm template kubearchive charts/kubearchive -n kubearchive \
     --include-crds \
-    --set "global.production=true" \
-    --set "database.enabled=false" > /tmp/kubearchive-not-resolved.yaml
+    --set "global.production=true" > /tmp/kubearchive-not-resolved.yaml
 ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > /tmp/kubearchive.yaml
 kubectl apply -n kubearchive -f /tmp/kubearchive.yaml
 
-cat <<EOF > /tmp/patch.yaml
+cat << EOF > /tmp/patch.yaml
```

Comment:
For temp file safety in bash, you should use `mktemp` rather than hard coding a temp file name. I suggest something like the below and then use the variable in future references.

```suggestion
PATCH_TMPFILE=$(mktemp --prefix patch --suffix .yaml)
cat << EOF > ${PATCH_TMPFILE}
```

Reply:
Sorry, I just realized you did not write this code. You were just updating the spacing. I will resolve this comment and submit a separate ticket to fix this.